Formula/p/proxygen.rb

```diff
@@ -43,9 +43,10 @@ def install
   end
 
   test do
-    pid = spawn bin/"proxygen_echo"
+    port = free_port
+    pid = spawn(bin/"proxygen_echo", "--http_port", port.to_s)
     sleep 10
```

Comment:
I am thinking failure may be timing related given only happening on Intel runners, which seem to getter slower.

Reply:
Doubling sleep to see if it helps. May be worth having some logic to increase any `sleep` for test retries as common reason retry passes is timing.